project-base/app/src/Model/Sitemap/SitemapListener.php

```diff
@@ -87,6 +130,48 @@ private function addUrlsForSitemapItems(
         }
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Sitemap\SitemapItem[] $sitemapItems
+     * @param \Presta\SitemapBundle\Service\AbstractGenerator $generator
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
+     * @param string $section
+     * @param string $routeName
+     * @param callable $getAlternativeForDomainCallable
+     */
+    private function addUrlsForSitemapItemsWithAlternativeLocations(
```

Comment:
why is this placed in the project-base implementation, please?

Reply:
moved to framework 👍